src/Components/Components/src/Reflection/MemberAssignment.cs

```diff
@@ -75,7 +75,7 @@ public void SetValue(object target, object value)
             {
                 if (value == null)
                 {
-                    _setterDelegate((TTarget)target, default);
+                    _setterDelegate((TTarget)target, default!);
```

Comment:
Is this because we can't annotate the underlying delegate?

Reply:
It's primarily because the property type could have been declared as non-nullable, but we don't have a good way to construct it. The runtime does this too for some generic methods: https://github.com/dotnet/runtime/blob/master/src/libraries/System.Linq/src/System/Linq/First.cs#L76